docker/config/besu/cliqueGenesis.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
why are those files new?

Reply:
I changed the integration test network from besu + orion to besu + tessera